src/Renderer/Label2DRenderer.js

```diff
@@ -0,0 +1,141 @@
+import * as THREE from 'three';
+
+const viewMatrix = new THREE.Matrix4();
+const viewProjectionMatrix = new THREE.Matrix4();
+const vector = new THREE.Vector3();
+const cameraSquaredPosition = new THREE.Vector3();
+let labels;
+let label;
+
+/**
+ * This renderer is inspired by the
+ * [`THREE.CSS2DRenderer`](https://threejs.org/docs/#examples/en/renderers/CSS2DRenderer).
+ * It is instanciated in `c3DEngine`, as another renderer to handles Labels.
+ */
+class Label2DRenderer {
+    constructor() {
+        this.domElement = document.createElement('div');
+        this.domElement.style.overflow = 'hidden';
+        this.domElement.style.position = 'absolute';
+        this.domElement.style.top = 0;
+
+        this.halfWidth = 0;
+        this.halfHeight = 0;
+
+        this.transform = '';
+
+        if (document.documentElement.style.transform !== undefined) {
+            this.transform = 'transform';
+        } else if (document.documentElement.style.webkitTransform !== undefined) {
+            this.transform = 'webkitTransform';
+        } else if (document.documentElement.style.mozTransform !== undefined) {
+            this.transform = 'mozTransform';
+        } else if (document.documentElement.style.oTransform !== undefined) {
+            this.transform = 'oTransform';
+        }
+    }
+
+    setSize(width, height) {
+        this.domElement.style.width = `${width}`;
+        this.domElement.style.height = `${height}`;
+
+        this.halfWidth = width / 2;
+        this.halfHeight = height / 2;
+    }
+
+    render(scene, camera) {
+        viewMatrix.copy(camera.matrixWorldInverse);
```

Comment:
why you copy `camera.matrixWorldInverse`? `viewMatrix` is useless?

Reply:
It is true, I'll fix it